css/style.css

```
@@ -0,0 +1,903 @@
+    transition: .3s;
+}
+
+.header_button__style:hover {
+    cursor: pointer;
+    background-color: rgb(103, 189, 192);
+    transition: .3s;
+    transform: scale(1.1);
+}
+
+.header_button__text {
+    font-family: Montserrat;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.12em;
+    color: #000000;
+    scroll-behavior: smooth;
+}
+
+.header_image {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 5%;
+    margin-top: -3.1%;
+}
+
+.main-content {
+    padding-top: 10.7%;
+    min-height: 985px;
+    background-color: #f8fafa;
+}
+
+.main-intro {
+    text-align: center;
+    margin-left: 23px;
+}
+
+.main-information {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 141px;
+}
+
+.main-intro__text-par {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.043em;
+    color: #000000;
+}
+
+.main-intro__text-content {
+    opacity: 0.61;
+    font-family: Merriweather;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 0.0699em;
+    color: #000000;
+    margin-top: 11px;
+    margin-left: -21px;
+}
+
+.first-par,
+.second-par {
+    opacity: 0.74;
+    font-family: Merriweather;
+    font-size: 20px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.068em;
+    color: #000000;
+}
+
+.first-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.07em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.second-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.first-par__text-second {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 0px;
+}
+
+.second-par {
+    letter-spacing: -0.036em;
+}
+
+.second-par_text {
+    margin-top: 13px;
+    margin-left: 1px;
+    letter-spacing: 0.0445em;
+}
+
+.main-image__style {
+    padding-left: 15px;
+}
+
+.main-information__text {
+    width: 474px;
+    padding-left: 2.3%;
+}
+
+.main-information__text-second {
+    margin-top: 29px;
+    margin-left: -2px;
+}
+
+.featured-works {
+    min-height: 1165px;
+    padding-top: 204px;
+}
+
+.featured-works__text {
+    text-align: center;
+    margin-left: 45px;
+}
+
+.featured-works__text-intro {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    color: #000000;
+}
+
+.featured-works__text-main {
+    opacity: 0.61;

```

I guess `0.6` is pretty enough :)

---

Thanks :))
I fixed)
```suggestion
opacity: 0.6;
```